datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVHelper.scala

```diff
@@ -147,6 +149,34 @@ object MVHelper {
     }.filter(_.isDefined).map(_.get)
   }
 
+
+  /**
+   * Check if we can do incremental load on the mv table. Some cases like aggregation functions
```

Comment:
A bit confused here, the function name is `isFullReload`, this function will return true if incremental load is not supported on input logicPlan?

Reply:
Yes, it checks whether a logical plan can support full refresh or incremental refresh.